webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc

```diff
@@ -0,0 +1,33 @@
+== *Create, Share and Consume Devfiles*
+:imagesdir: ../../../images
+'''''
+The following is an example of a devfile ecosystem that is framed within the context of an organization looking to standardize their development environment.  In the simplest case, developers can just consume the devfiles that are available from the public community registry.  If the organization sees the need to author and share their own devfiles internally, they will need a role based approach where Developers, Devfile Authors, and Registry Administrators can interact together.
+
+image::devfileEcosystem.svg[]
+
+
+=== *The Main Areas of the Devfile Ecosystem*
+=== *Create*
+
+A devfile author aka “stack provider” can be an individual or a group representing a runtime vendor.  This role requires sound knowledge of the supported runtime so devfiles can be created with the proper guidance on how to build and run applications.
+
+If a runtime stack is not available in the public registry, an organization can choose to develop their own and keep it private for their in-house development.
+
+=== *Share*
+
+The public community registry is managed by the community and hosted by Red Hat.  This is one way to share your devfile if there is a benefit to the greater community.
```

Comment:
For the second sentence, I'd put the action on the user. I think that'd motivate them more to work with devfiles. What do you think of this rephrase?:

`Share your devfile to the public community registry so other teams can benefit from your application.`

Reply:
done